apps/mark-scan/fai-100-controller/src/usb_device.rs

```diff
@@ -0,0 +1,114 @@
+use std::{io, time::Duration};
+
+use color_eyre::eyre::Result;
+use rusb::{Context, DeviceHandle, UsbContext};
+
+const INTERFACE_NUMBER: u8 = 0x00;
+const CONFIG_NUMBER: u8 = 0x01;
+const SETTING_NUMBER: u8 = 0x00;
+const ENDPOINT_ADDRESS_IN: u8 = 0x81;
+const ENDPOINT_ADDRESS_OUT: u8 = 0x01;
```

Comment:
Per conversations with the manufacturer `0x81` and `0x01` are fixed and can be hard coded.